sirepo/package_data/static/js/sirepo-components.js

```diff
@@ -1871,6 +1871,51 @@ SIREPO.app.directive('validatedString', function(panelState, validationService)
     };
 });
 
+SIREPO.app.directive('viewLogIframe', function() {
+    return {
+        restrict: 'A',
+        scope: {
+            loadingLog: '<',
```

Comment:
How about logIsLoading? "is" implies a boolean state which helps tip off readers that's what this is used for.

Reply:
4d8ecf81fb4e98997af2c3b6de70315095700cc9